README.md

```diff
@@ -10,4 +10,8 @@ Describes how to download, build, and install Debian on the Poplar board.
 
 - [Android](android/)
 
-Describes how to download, build, and install Android on the Poplar board.
\ No newline at end of file
+Describes how to download, build, and install Android on the Poplar board.
+
+- [Recovery](recovery.md)
+
+Describes how to do a board recovery on the Poplar board.
```

Comment:
Maybe call it debain_recovery.md? And create a directory call Debian (in parallel with  Android) and put the (debain_)recovery.md and build_debain_system.md there?

Reply:
Maybe it doesn't apply as much to Android if not developing for bootloader, but recovery can be used for both Android and Debian.

Reply:
Ok for creating dir called `Debian` and putting `debian_build_instructions.md` there if that's preferred?

Reply:
Since prebuilt recovery files are Android-specific, this makes `recovery.md` Debian-specific, so ok, I'll create a `Debian` dir and move the 2 files into it.